extension/ref-inject-code-search.js

```diff
@@ -29,8 +40,8 @@ var _RESOURCE_MAP = {
   'R.array'         : 'values/arrays.xml',
   'R.attr'          : 'values/attrs.xml',
   'R.bool'          : 'values/bools.xml',
-  'R.color'         : 'values/colors_material.xml',
-  'R.dimen'         : 'values/dimens_material.xml',
+  'R.color'         : 'values/colors.xml',
```

Comment:
ah, that's a good point. in that case i'd prefer we special-case `android.R.color` to point to colors_material and everything else can point to colors